wolvic/browser/vr/wvr_manager.cc

```diff
@@ -305,20 +307,22 @@ void WvrManager::OnWebXrFrameAvailable() {
   if (!webxr_frame_timeout_closure_.IsCancelled())
     webxr_frame_timeout_closure_.Cancel();
 
-  // Frame should be locked. Unlock it.
-  DCHECK(webxr_->GetProcessingFrame()->state_locked);
-  webxr_->GetProcessingFrame()->state_locked = false;
+  if (webxr_->HaveProcessingFrame()) {
```

Comment:
I believe we need a comment here briefly stating why this check is needed because it is not obvious taking into account that we're already in `OnWebXrFrameAvailable`

Reply:
Done